source/daplink/drag-n-drop/intelhex.c

```diff
@@ -178,10 +178,6 @@ hexfile_parse_status_t parse_hex_blob(const uint8_t *hex_blob, const uint32_t he
                                             // Save next address to write
                                             next_address_to_write = ((next_address_to_write & 0xffff0000) | line.address) + line.byte_count;
                                         }
-                                        else {
-                                          status = HEX_PARSE_OK;
-                                          goto hex_parser_exit;
-                                        }
```

Comment:
This increased the flashing time of a Universal Hex by about one second with this example hex: [uhex-blocks-upy.hex.zip](https://github.com/ARMmbed/DAPLink/files/7197282/uhex-blocks-upy.hex.zip)
Mostly the time to parse content for a different target (micro:bit V1 data) goes up from ~3s to ~4s.

Reply:
This part is required to be able to flash the `uhex-sectors-upy.hex` file I was under the impression needed to be supported.